.eslintrc.js

```diff
@@ -0,0 +1,102 @@
+const generateColorsRules = () => ({
```

Comment:
No need to move this into a js file, you only have hardcoded rules :)

Reply:
Also be careful you overwrite the no-restricted-syntax rule https://github.com/scaleway/scaleway-lib/blob/d0e138501ae3c7fb59b8e5924504e8cef434070e/packages/eslint-config-react/shared.js#L25